concepts/packages/about.md

```diff
@@ -52,3 +52,18 @@ errors.New("Connection not established")
 // using the errors package located at github.com/pkg/errors by alias
 errs.New("Connection not established")
 ```
+
+Similar to program languages like Java that use access modifiers such as `public` and `private`, Go determines if an item is exported and unexported through how it is declared. 
```

Comment:
The term "similar" was a bit confusing for me because in the end its not very similar when you look at the syntax.
```suggestion
Languages like Java use access modifiers such as `public` and `private` to specify what is accessible from outside. Go determines whether an entity is exported from a package or not through how it is declared.
```